File: dayOne/java/levelThree/MergeSortArray.java
public class MergeSortArray {

    public static int[] mergeArray(int[] arrayOne, int[] arrayTwo) {
        int length = arrayOne.length + arrayTwo.length;
        int[] mergedArray = new int[length];
        int count = 0;

        for (int element : arrayOne)
            mergedArray[count++] = element;

        for (int element : arrayTwo)
            mergedArray[count++] = element;


        for (int index = 0; index < length; index++) {
            int currentIndex = index;

            for (int innerIndex = index + 1; innerIndex < length; innerIndex++) {
                if (mergedArray[innerIndex] < mergedArray[currentIndex])
                    currentIndex = innerIndex;
            }

            if (index != currentIndex) {
                int temporaryValue = mergedArray[index];
                mergedArray[index] = mergedArray[currentIndex];
                mergedArray[currentIndex] = temporaryValue;
            }
        }

        return mergedArray;
    }
}

File: dayOne/python/levelThree/merge_sort_array.py
def merge_sort(listOne, listTwo):
    new_list = listOne + listTwo

    for index in range(len(new_list)):
        current_index = index

        for inner_index in range(index + 1, len(new_list)):

            if new_list[current_index] > new_list[inner_index]:
                current_index = inner_index

        if current_index != index:
            temporary_value = new_list[index]
            new_list[index] = new_list[current_index]
            new_list[current_index] = temporary_value

    return new_list
